wordle-web/src/components/NavigationBar.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-card>
+    <v-layout>
+      <v-app-bar>
+        <template v-slot:image> </template>
+        <template v-slot:prepend>
+          <router-link to="/"
+            ><v-app-bar-nav-icon icon="mdi-vuejs"></v-app-bar-nav-icon
+          ></router-link>
+          <router-link to="/"
+            ><v-app-bar-title class="primary" color="primary"> wordle</v-app-bar-title></router-link
+          >
+        </template>
+        <v-spacer></v-spacer>
+
+        <v-btn>
+          <SettingsDialogVue />
+        </v-btn>
+
+        <template v-slot:append>
+          <v-menu>
+            <template v-slot:activator="{ props }">
+              <v-btn v-bind="props" icon="mdi-hamburger"> </v-btn>
```

Comment:
Is not visible, Background takes over

Reply:
Hi Josh, does the background take over when you scroll up?